network-store-client/src/main/java/com/powsybl/network/store/client/TerminalBusViewImpl.java

```diff
@@ -8,18 +8,22 @@
 
 import com.powsybl.iidm.network.Bus;
 import com.powsybl.iidm.network.Terminal;
+import com.powsybl.network.store.model.InjectionAttributes;
 
 import java.util.Objects;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-class TerminalBusViewImpl implements Terminal.BusView {
+class TerminalBusViewImpl<U extends InjectionAttributes> implements Terminal.BusView {
 
     private final NetworkObjectIndex index;
 
-    TerminalBusViewImpl(NetworkObjectIndex index) {
+    private final U attributes;
```

Comment:
attributes is never used. Is it planned to be used in future devs ?

Reply:
yes